app/src/test/java/org/mozilla/fenix/share/ShareCloseViewTest.kt

```diff
@@ -26,18 +26,19 @@ class ShareCloseViewTest {
 
     @Before
     fun setup() {
-        container = FrameLayout(testContext)
+        container = ConstraintLayout(testContext)
```

Comment:
nit: Is this change needed?

Reply:
Yes.

![image](https://user-images.githubusercontent.com/60002907/129716209-c67c3d95-658d-491a-a360-c94859f38ddd.png)